hw/ip/flash_ctrl/doc/_index.md

```diff
@@ -290,6 +302,23 @@ However, one TBD feature is related to flash support of life cycle and manufactu
 It may be required for manufacturers to directly inject data into specific pages flash information partitions via die contacts.
 For these pages, scramble shall be permanently disabled as the manufacturer should not be aware of scrambling functions.
 
+## Flash Default Configuration
+Since the flash controller is highly dependent on the specific flavor of flash memory chosen underneath, its configuration can vary widely between different integrations.
+
+This sections details the default settings used by the flash controller:
+* Number of banks: 2
+* Number of pages per bank: 256
+* [Program resolution](#program-resolution): 8 flash words
+* Flash word data bits: 64
+* Flash word metadata bits: 8
+* ECC choice: Hamming code SECDED
+* Information partition types: 3
+* Size of information partition type 0: 10 pages
+* Size of information partition type 1: 1 page
+* Size of information partition type 2: 2 pages
```

Comment:
As I recall, the type 1 & 2 partitions are for internal management of the flash macro and shouldn't be considered as partitions that we can actually use.

Reply:
yes but that's not a requirement from the open source side. So if a particular flash actually made multiple available, or allowed to re-use them in someway, at least it will be an available option.